ember-cli-build.js

```diff
@@ -3,52 +3,35 @@
 'use strict'
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app')
+const Funnel = require('broccoli-funnel')
 
-module.exports = function (defaults) {
+module.exports = function(defaults) {
   let app = new EmberApp(defaults, {
     sassOptions: {
       onlyIncluded: true
     },
-    postcssOptions: {
-      compile: { enabled: false },
-      filter: {
-        enabled: true,
-        plugins: [
-          {
-            module: require('postcss-cssnext'),
-            options: {
-              features: {
-                customProperties: {
-                  warnings: false
-                }
-              }
-            }
-          }
-        ]
-      }
-    },
     babel: {
-      plugins: [
-        'transform-async-to-generator',
-        'transform-decorators-legacy',
-        'transform-object-rest-spread'
-      ]
+      plugins: ['@babel/plugin-proposal-object-rest-spread']
     },
     'ember-site-tour': {
       importHopscotchJS: true,
       importHopscotchCSS: true
     }
   })
 
+  app.import('node_modules/typeface-source-sans-pro/index.css')
   app.import('vendor/adcssy.min.css')
 
   app.import('node_modules/downloadjs/download.min.js', {
```

Comment:
This can be imported via `ember-auto-import`, remove this

Reply:
Removed it. There's probably more that can be removed here.

Specifically, I intend to switch from the self-hosted font to Google Fonts. And for downloadjs I'll probably switch to [FileSaver](https://github.com/eligrey/FileSaver.js).

Reply:
Right, I agree.. Let's do that in another PR after this one is through